src/gpl/CMakeLists.txt

```diff
@@ -85,6 +85,32 @@ target_link_libraries(gpl
     rsz
     grt
 )
+# If CUDA is not installed there will not be cmake
+# support for the package so this needs to be "quiet".
+find_package(CUDA QUIET COMPONENTS Core Widgets)
+
+# Allow users to use GPU or not
+option(GPU "Enable GPU" OFF)
+
+if (CUDA_FOUND AND GPU)
```

Comment:
If GPU and !CUDA_FOUND warrants a warning beyond the status "GPU is not enabled"